performance-tests/perf-test-commits.sh

```diff
@@ -0,0 +1,113 @@
+#! /usr/bin/env bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+# perf-test-commits is intended to generate a comparable performance tests results across a series of commits.
+# It will checkout each commit in turn and build an image for each commit before running `pref-test.sh` with each generated image.
+# By building multiple images and running them in a single session we ensure identical hardware (and thus performance characteristics)
+# to hopefully allow us to isolate the performance impact of each commit.
+PERF_TESTS_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+source "${PERF_TESTS_DIR}"/../scripts/common.sh
+
+GREEN='\033[0;32m'
+NOCOLOR='\033[0m'
+
+FIND_COMMAND="find"
+if [ "$OS" = 'Darwin' ] && resolveCommand gfind ; then
```

Comment:
```suggestion
if [ "$OS" = 'Darwin' ] && resolveCommand gfind 1>- ; then
```

Reply:
My bash and google foo are struggling with this one. https://www.gnu.org/software/bash/manual/html_node/Redirections.html#Moving-File-Descriptors seems to imply that it would redirect `STD_OUT` to `STD_OUT`

Reply:
@robobario suggested that it was trying to make `gfind` optional. Which is a good point which I've tried to address with https://github.com/kroxylicious/kroxylicious/pull/1207/commits/a9f98abb543210a0d490004eba5717beb856b678

Reply:
I screwed up.  My intent was to discard the output from shell if `gfind` wasn't found.  It was an inappropiateattempt to close a descriptor and I used the wrong syntax.  I ought to have written 1>/dev/null.

Reply:
I agree that keeping gfind optional is best, so +1 to a change like Rob's